### PR TITLE
Release 0.8.2

### PR DESCRIPTION
**Pull request by stefanor**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+0.8.2
+-----
+
+* `DjangoApp`: Correctly detect Django version used. Until now yodeploy
+  has incorrectly assumed >= 1.7, when performing migrations.
```
Comment (beck) [inline CHANGELOG.md]: I kinda like it when changelogs link to the pull that landed the change.

Comment (stefanor) [inline CHANGELOG.md]: I hate those, I find it adds a lot of noise to the changelogs.

Comment (stefanor) [inline CHANGELOG.md]: I can always git blame to find the change. esp if the changelog entry was in the PR that made the change.

**Pull request merged**